src/Sulu/Bundle/ContentBundle/Command/WebspaceCopyCommand.php

```diff
@@ -0,0 +1,675 @@
+ */
+class WebspaceCopyCommand extends ContainerAwareCommand
+{
+    /**
+     * @var OutputInterface
+     */
+    private $output;
+
+    /**
+     * @var DocumentManager
+     */
+    private $documentManager;
+
+    /**
+     * @var SessionManagerInterface
+     */
+    private $sessionManager;
+
+    /**
+     * @var DocumentInspector
+     */
+    private $documentInspector;
+
+    /**
+     * @var HtmlTagExtractor
+     */
+    private $htmlTagExtractor;
+
+    /**
+     * @var string
+     */
+    protected $webspaceKeySource;
+
+    /**
+     * @var string
+     */
+    protected $webspaceKeyDestination;
+
+    protected function configure()
+    {
+        $this->setName('sulu:webspaces:copy')
+            ->addArgument('source-webspace', InputArgument::REQUIRED)
+            ->addArgument('source-locale', InputArgument::REQUIRED)
+            ->addArgument('destination-webspace', InputArgument::REQUIRED)
+            ->addArgument('destination-locale', InputArgument::REQUIRED)
+            ->addOption('clear-destination-webspace')
+            ->setDescription('Copy whole webspace');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $webspaceKeySource = $input->getArgument('source-webspace');
+        $localesSource = explode(',', $input->getArgument('source-locale'));
+        $webspaceKeyDestination = $input->getArgument('destination-webspace');
+        $localesDestination = explode(',', $input->getArgument('destination-locale'));
+
+        if (count($localesSource) !== count($localesDestination)) {
+            $output->writeln([
+                '<error>Aborted!</error>',
+                '<comment>Provide correct source and destination locales</comment>',
+            ]);
+
+            return -1;
+        }
+
+        $output->writeln([
+            '<info>Webspace Copy</info>',
+            '<info>==============================</info>',
+            '',
+            '<info>Options</info>',
+            '------------------------------',
+            'Webspace: ' . $webspaceKeySource . ' => ' . $webspaceKeyDestination,
+        ]);
+
+        $localesPairs = [];
+        for ($i = 0; $i < count($localesSource); ++$i) {
+            $localesPairs[] = $localesSource[$i] . ' => ' . $localesDestination[$i];
+        }
+        $output->writeln([
+            'Locales: ' . implode(', ', $localesPairs),
+            '------------------------------',
+            '',
+        ]);
+
+        $output->writeln([
+            '<info>==============================</info>',
+            '<info>ATTENTION</info>',
+            '<info>The whole destination webspace (' . $webspaceKeyDestination . ') will be deleted!</info>',
+            '<info>==============================</info>',
+            '',
+        ]);
+
+        if (true !== $input->getOption('clear-destination-webspace')) {
+            $output->writeln([
```

Comment:
This were the methods I've been talking about: http://symfony.com/doc/current/console/style.html#result-methods

Then the style would be nicer and more consistent.